blockchains/erc20/erc20_worker.js

```diff
@@ -29,6 +29,21 @@ class ERC20Worker extends BaseWorker {
     if (constants.EXPORT_BLOCKS_LIST) {
       this.blocksList = await initBlocksList();
     }
+    console.log(JSON.stringify(constants));
+    if (constants.CONTRACT_MODE !== 'vanilla') {
+      const parsedContracts = await readJsonFile(constants.CONTRACT_MAPPING_FILE_PATH);
+
+      if (parsedContracts.modified_contracts) {
+        this.contractsOverwriteArray = parsedContracts.modified_contracts.map((contract) => new ContractOverwrite(contract));
+      }
+      if (parsedContracts.unmodified_contracts) {
```

Comment:
That's the new functionality, where we extract per contract, without overwriting.